charts/gardener/charts/utils-common/templates/_secret-alerting.yaml

```diff
@@ -0,0 +1,27 @@
+{{- define "gardener.secret-alerting-smtp" -}}
+{{- if .Values.global.controller.enabled }}
+{{- range $key, $config := .Values.global.controller.alerting }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: alerting-smtp-{{ $key }}
+  namespace: garden
+  labels:
+    app: gardener
+    chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
+    release: "{{ $.Release.Name }}"
+    heritage: "{{ $.Release.Service }}"
+    gardener.cloud/role: alerting
+type: Opaque
+data:
+  auth_type: c210cA==
+  to: {{ ( required ".controller.alerting[].to is required" $config.to ) | b64enc }}
+  from: {{ ( required ".controller.alerting[].from is required" $config.from ) | b64enc }}
+  smarthost: {{ ( required ".controller.alerting[].smarthost is required" $config.smarthost ) | b64enc }}
+  auth_username: {{ ( required ".controller.alerting[].auth_username is required" $config.auth_username ) | b64enc }}
+  auth_identity: {{ ( required ".controller.alerting[].auth_identity is required" $config.auth_identity ) | b64enc }}
+  auth_password: {{ ( required ".controller.alerting[].auth_password is required" $config.auth_password ) | b64enc }}
```

Comment:
Why are the other keys not configurable via the Helm chart? I see many more possible keys in the example alerting secret.